src/sources/invictus.ts

```diff
@@ -28,19 +28,24 @@ export class StakedInvictusMarketSource implements MarketSource {
    *
    * @return Array containing one element which is sIN
    */
-  async query(invictusPrice: number): Promise<MarketDataMap> {
+  async query(marketDataMap: MarketDataMap): Promise<MarketDataMap> {
```

Comment:
This change should have been applied to all `MarketSource` types and the interface.